bundle/bundle.go

```diff
@@ -555,7 +556,7 @@ func (r *Reader) Read() (Bundle, error) {
 				return bundle, err
 			}
 
-		} else if filepath.Base(path) == yamlDataFile {
+		} else if filepath.Base(path) == yamlDataFile || filepath.Base(path) == ymlDataFile {
```

Comment:
@Joffref thanks for the contribution. Can you please add a test case.

Reply:
Done !